src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const obj = { ...state };
+  const result = [];
+
+  for (const action of actions) {
+    // якщо key.type === 'addProperties'
```

Comment:
Use a destruction for action
```suggestion
  for (const action of actions) {
    const {} = action;
```